app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Constraints.scala

```diff
@@ -94,14 +94,13 @@ trait Constraints {
         Valid
     }
 
-  protected def nonEmptySet(errorKey: String): Constraint[Set[_]] =
+  protected def nonEmptySet(errorKey: String): Constraint[Set[_]]                                       =
```

Comment:
Not sure what you did here. Seems that you added white space only. Please remove if this is so.

Reply:
That was done by scalafmtAll function.